014/master

berziiii opened the pull request:
Update from template and address various open issues.

Commit:
Update editorconfig and gitignore from template

-Update editor config to match template
-Update gitignore to add changes from template

Commit:
Finalize js-template update

-Update from JS-template using reset and add --patch to ensure information is
not lost

Commit:
Correct file name in package.json

-Update name to 'ga-wdi-boston.html-css'

Commit:
Add git rebase to update css branch

- address issue #20 to udpate `css` branch with the changes added to `master` after
we merged the `html` branch into `master`

Commit:
Correct linter errors

-Fix linter errors for various issues

jrhorn424 commented on README.md:
```diff
@@ -229,6 +229,17 @@ the `main.css` file in the `assets/styles/` directory as follows. Don't worry
+rebase. We will learn more about what a rebase does in unit 3, so don't worry
+about _understanding_ it right now, but to get our `css` branch up to date with
+master run (double check to confirm that you've checked out the `css` branch)
+`git rebase master`. **IF** you have issues, flag down a consultant
```
Does this make more sense as a bulleted list of instructions?

```
about _understanding_ it right now. To get our `css` branch up to date with `master`:

-   Confirm you've checked out the `css` branch with `git status`.
-   Update the `css` branch with `git rebase master`.
-   If you experience any issues, flag down a consultant.

Now, in your `main.css` file:
```

jrhorn424 commented:
Looks good! What do you think of my suggestions? @berziiii

Commit:
Clarify rebase portion of README.md

- Bullet the rebase portion once html branch has been merged into master, before
css, to help with clarity and ensure steps are followed

berziiii commented:
Yes, changes made. Thanks for suggestion!
